searx/engines/__init__.py

```diff
@@ -79,16 +79,18 @@ def load_engine(engine_data):
         logger.exception('Cannot load engine "{}"'.format(engine_module))
         return None
 
-    for param_name in engine_data:
+    for param_name, param_value in engine_data.items():
         if param_name == 'engine':
-            continue
-        if param_name == 'categories':
-            if engine_data['categories'] == 'none':
+            pass
```

Comment:
`pass`? Why not just remove this branch of the `if` if nothing has to happen in case of engines?

Reply:
By default, all attributes are copied to `engine`, see line 92 and 93. 
This line says explicitly that the `engine` attribute name is skipped.